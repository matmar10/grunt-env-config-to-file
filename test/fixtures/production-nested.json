{
  "someOtherProperty": "someOtherValue",
  "level0": {
    "level0Prop": "level0Value",
    "level1": {
      "level1Prop": "level1Value",
      "level2": {
        "level2Prop": "level2Value",
        "level3": {
          "level3Prop": "level3Value"
        }
      }
    }
  }
}
