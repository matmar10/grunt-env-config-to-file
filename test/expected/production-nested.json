{
  "someOtherProperty": "someOtherValue",
  "level0": {
    "level0Prop": "level0Value",
    "level1": {
      "level1Prop": "level1Value",
      "level2": {
        "level2Prop": "level2Value",
        "level3": {
          "level3Prop": "level3Value"
        },
        "RABBITMQ_BIGWIG_URL": "amqp://user:pass@host:10000/vhost"
      }
    },
    "DATABASE_URL": "postgresql://user:pass@host:port/database"
  }
}